pkg/infoschema/tables.go

```diff
@@ -232,6 +232,13 @@ const (
 	DataLockWaitsColumnSQLDigestText = "SQL_DIGEST_TEXT"
 )
 
+const (
+	// TSOServiceName is the name of TSO service.
+	TSOServiceName = "tso"
+	// SchedulingServiceName is the name of scheduling service.
+	SchedulingServiceName = "scheduling"
```

Comment:
Seems no need to be exported.

Reply:
Do we need to add more comments to indicate they are related to PD Micro Services?

Reply:
done